src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -1078,13 +1079,54 @@ else if (block instanceof BlockFlowerPot)
             }
         }
 
-        if (block instanceof BlockChest && job instanceof JobBuilder)
-        {
-            final BlockPos buildingLocation = ((JobBuilder) job).getWorkOrder().getBuildingLocation();
-            final AbstractBuilding building = this.getOwnBuilding().getColony().getBuilding(buildingLocation);
-            building.addContainerPosition(pos);
+
+        try{
+            if (job instanceof JobBuilder) {
```

Comment:
That should be away already I think. I rewrote all that builder stuff already.
Probably just has to merge the newest changes to his branch